cms/templates/partials/hero.html

```diff
@@ -40,13 +40,13 @@ <h2>{{ page.subhead }}</h2>
         <div class="mt-5 mb-5">
           {% if page.about_mit_xpro.video_url and action_title and action_url %}
             <a id="actionButton" class="btn btn-primary text-uppercase px-5 py-2 action-button" href="{{ action_url }}">{{ action_title }}</a>
+          {% elif page.is_external_course_page %}
+            <a class="enroll-button enroll-now" href="{{ page.external_url }}">
+              Apply Now
```

Comment:
would it be shown up for anonymous user as well ?

Reply:
Yes, @pdpinch confirmed it should be shown always.